libcontainer/cgroups/utils.go

```diff
@@ -21,49 +21,31 @@ const cgroupNamePrefix = "name="
 
 // https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt
 func FindCgroupMountpoint(subsystem string) (string, error) {
-	f, err := os.Open("/proc/self/mountinfo")
+	mounts, err := mount.GetMounts()
```

Comment:
All right, then I think we should keep it efficient, but it breaks freebsd, maybe freebsd should offer an get cgroup mount point function, we can deal with that later.
I'll update this, thanks.